src/Razor/src/Microsoft.VisualStudio.LanguageServices.Razor/Logging/ActivityLogLoggerProvider.Logger.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT license. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.AspNetCore.Razor.PooledObjects;
+using Microsoft.CodeAnalysis.Razor.Logging;
+using Microsoft.VisualStudio.Razor.Logging;
+
+namespace Microsoft.VisualStudio.LanguageServices.Razor.Logging;
+
+internal sealed partial class ActivityLogLoggerProvider
+{
+    private sealed class Logger(RazorActivityLog activityLog, string categoryName) : ILogger
+    {
+        private readonly RazorActivityLog _activityLog = activityLog;
+        private readonly string _categoryName = categoryName;
+
+        public bool IsEnabled(LogLevel logLevel)
+            => logLevel is LogLevel.Warning or LogLevel.Error or LogLevel.Critical;
```

Comment:
```suggestion
            => logLevel >= LogLevel.Warning
```

Reply:
I specifically didn't go that route because it allows `LogLevel.None`, which didn't seem like a legal value to me.

Reply:
https://source.dot.net/#Microsoft.Extensions.Logging.Abstractions/LogLevel.cs,d07793e8b722b77e and https://github.com/dotnet/razor/blob/main/src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/Logging/LogLevel.cs

Reply:
oof... whyyyyyyyyy

Reply:
I have no idea what that's about, but that's why I was nervous.

Reply:
We have the power... 

Also, wonder if it should just be some weird extension method like `logLevel.IsAtLeast(LogLevel.Warning)` where it checks for `None` explicitly. 

Otherwise our `None` could be 0 instead of 6

Reply:
That's a good idea.

Reply:
We could just remove None. I thought it was weird when I copied it from MS.Ext.Logging, but didn't want to rock the boat when I was OOF. I think we just need to do it in the settings in VS Code too. I doubt we use it.

Reply:
I think we want None. We want there to be a log level that disables everything.

Reply:
Also, I don't know that I'm comfortable with moving None to be a different value, since that requires changes and assumption validation elsewhere. Adding an extension method as @ryzngard suggested seems like the approach that's more in line with the scope of this PR.

Reply:
> We want there to be a log level that disables everything.

Oh, derp, of course. 🤦‍♂️